README.md

```diff
@@ -227,6 +227,41 @@ firestoreService.patch('regions/32/users/1', body);
 
 An error will be thrown because of id mismatch.
 
+
+## Authentications
+
+Firestore service also supports methods to authenticate users. These are not http methods, are functions to be used with the same package.
+
+### LOGIN
+
+You can login with an already created user (You can create them in the firebase console).
+
+```js
+const email = 'email@example.com'
+const password = 'xxxxxx'
+
+
+const response = await firestoreService.login(email, password);
+```
+
+The response will have all the user info.
+
+### SIGN UP
+
+You can sign up a new user with the sign up method.
+
+```js
+const email = 'email@example.com'
+const password = 'xxxxxx'
+
+
+const response = await firestoreService.signUp(email, password);
+```
+
+If the response is succesful, the user will be created and all it's information will be in the response.
```

Comment:
maybe we should add a response example

Reply:
The user info response is full of information, it would be quite hard to add an example of it